_config.yml

```diff
@@ -206,6 +206,10 @@ component_releases:
 release_archive:
     flink:
       - version_short: "1.12"
+        version_long: 1.12.2
+        release_date: 2021-02-16
```

Comment:
Please remember to update the release date accordingly.

Reply:
I'll update the date before merging. Thanks!